MultiTable/src/Utils.js

```diff
@@ -28,7 +28,21 @@ module.exports = class Utils {
       subjects: {},
     };
     if (subject.absolutePath) {
-      inv.subjects[subject.absolutePath.toLowerCase()] = subject;
+
+      // deep copy to avoid modifying the input
+      const keys = Object.keys(subject);
```

Comment:
this 'copy only the necessary keys' is faster than deep copy the entire thing. Perf test: https://jsperf.com/deepcopyremovekey

Reply:
I agree that this is better than deep copying the whole thing.
It would be simpler to just do a forEach and check the key name on each iteration.